admin/views/privacy.ejs

```diff
@@ -0,0 +1,117 @@
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-third">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <h1 class="heading-xlarge">Privacy notice</h1>
+        <ul class="list list-contents">
+            <li>Contents</li>
+            <li class="list-item--dashed"><a href="#overview">Overview</a></li>
+            <li class="list-item--dashed"><a href="#data-collection-and-access">What data will be collected and who can access it?</a></li>
+            <li class="list-item--dashed"><a href="#retaining-personal-data">Retaining personal data</a></li>
+            <li class="list-item--dashed"><a href="#further-information">Further information</a></li>
+        </ul>
+        <hr>
+        <div id="overview">
+            <h3 class="heading-medium">Overview</h3>
+            <p>The Scottish Qualification Authority (SQA) is administering the multiplication tables check (MTC) technical
+            pre-test trials on behalf of the Standards and Testing Agency (STA).
+            The trials will take place in a sample of schools from 25 June to 6 July 2018.</p>
+            STA will use the results of the trial to:
+            <p>
+            <ul class="list list-bullet">
+                <li>create a range of check forms (online question papers) to ensure
+                    they are of comparable difficulty for implementing in national checks from 2020
+                </li>
+                <li>develop and refine the digital service for implementing live checks</li>
+            </ul>
+            STA is the data controller for the trials and is responsible for determining how personal data collected
+            during the period will be treated.
+            As with all trials of new national assessments, we will not publish the results of the check, and no
+            individual school or pupil will be identified.
+            </p>
+            <p>As data processor, SQA is responsible for securely storing and handling any personal data required to
+            administer the trials.
+            They will securely dispose of this data at the end of the MTC contract, in accordance with the agreement
+            with STA, and in line with the requirements of UK data protection legislation and the
+            <a target="_blank" href="https://ico.org.uk/for-organisations/guide-to-the-general-data-protection-regulation-gdpr">
+                General Data Protection Regulation (GDPR)
+            </a>.&nbsp<img src="/images/icon-link.png"/></p>
+        </div>
+        <p id="data-collection-and-access">
+            <h3 class="heading-medium">What data will be collected and who can access it?</h3>
```

Comment:
<h> tags should be outside of <p> tags- can you check this html is valid please?